src/app/modules/submission/encrypt-submission/encrypt-submission.utils.ts

```diff
@@ -210,3 +219,104 @@ export const createEncryptedSubmissionDto = (
     attachmentMetadata: attachmentPresignedUrls,
   }
 }
+
+export class IncomingEncryptSubmission extends IncomingSubmission {
+  constructor(
```

Comment:
Would it be good to make this constructor private to make sure that `.init` is called i.e. prevent direct calls to the constructor by accident?

Reply:
thanks for the review! good idea, will make this private